StellarDotnetSdk/Responses/SorobanRpc/GetHealthResponse.cs

```diff
@@ -1,18 +1,33 @@
-﻿using Newtonsoft.Json;
```

Comment:
I thought in the last PR we already decided to switch to `System.Text.Json` right? Or do I remember this incorrectly?

Reply:
Yes, you got it right, but my attempt to implement the custom converters with System.Text.Json didn't go really well, so I've decided to stick with Json.Net. I thought I told you about this.

Reply:
I thought the improvement we had in mind was not possible but was it not possbile to solve it in another way and then still use the system.text.json?

Reply:
Alright doing it in a new PR